x-pack/plugins/cases/public/components/case_view/index.test.tsx

```diff
@@ -274,38 +280,44 @@ describe('CaseView', () => {
   });
 
   it('should refresh data on refresh', async () => {
+    const queryClient = new QueryClient();
+    const queryClientSpy = jest.spyOn(queryClient, 'invalidateQueries');
     const wrapper = mount(
       <TestProviders>
-        <CaseView {...caseViewProps} />
+        <QueryClientProvider client={queryClient}>
```

Comment:
It doesn't matter if you wrap it twice, it will use the closest one to the component. QueryClientProvider behaves as a regular react context provider.

Using the AppMockRenderer is another way, just a matter of implementation it doesn't change the outcome in anyway.

Reply:
Ok! Thanks for clarifying.